### ISSUE TITLE
asizeof get wrong size with numpy arrays after slicing

### ISSUE THREAD

**Issue by sinorga** [pympler/pympler#111]
Its similar with #36

Environment: 
ubuntu 1804, 64 bit
python 3.7.5
numpy 1.17.3
Pympler 0.8

----

```
>>> import numpy as np
>>> from pympler import asizeof
>>> arr = np.arange(1000)
>>> asizeof.asizeof(arr)
8096
>>> arr = arr[:]
>>> asizeof.asizeof(arr)
96
```
The size is expected to the same.

**Comment by mrJean1** [comment]
What is the `asizeof.__version__`?

**Comment by sinorga** [comment]
It's `19.03.29`

**Comment by mrJean1** [comment]
Thank you.  So far, I'm able to duplicate the issue with asizeof 19.03.29 using Python 2.7.18 and numpy 1.16.5, with Python 3.7.6 and numpy 1.17.2 and with Python 3.8.2 and numpy 1.18.0, all in 64-bit and on macOS 10.13.6.  Investigating the root cause.

**Comment by mrJean1** [comment]
After the patch **Handle NumPy slices**, the `asizeof` slice `b` will include and be slightly larger than the size of `a`.  Likewise, the combined `asizeof(a, b)` will just be the size of `b`.

**Comment by Pankrat** [comment]
Thanks @sinorga and @mrJean1! Your fix has been included in v0.9.
